strider/trapi_throttle/trapi.py

```diff
@@ -66,7 +66,7 @@ def result_contains_node_bindings(
     """ Check that the result object has all bindings provided (qg_id->kg_id) """
     for qg_id, kg_ids in bindings.items():
         if not any(
-            nb["id"] in kg_ids
+            nb["id"] in kg_ids or nb.get("qnode_id") in kg_ids
```

Comment:
Where is this `qnode_id` coming from? And is there a reason we combine the direct access and get method when looking for these values?

Reply:
This `qnode_id` business is an experimental feature for enabling unambiguous unbatching. It has been implemented by Automat. We use the direct access for `id` because it is a required field and we know it'll be there, but we use `get()` for `qnode_id` because most bindings will not have it - we'll get None instead of a KeyError.